content/docs/integrations/integrations-list.md

```diff
@@ -1,22 +1,25 @@
 ---
 title: Integrations
+redirectFrom:
+  - docs/cloud/integrations
 ---
 
```

Comment:
Can we add a short introductory sentence, such as:
"This page lists the available Neon integrations."